### DISCUSSION TITLE
How to implement monitoring for the remote server when using Multi Server feature

### DISCUSSION THREAD

**Discussion by MuhammadM1998** [Dokploy/dokploy#1015] [Q&A] [answered]
Hello 👋

The [Multi Server](https://docs.dokploy.com/docs/core/multi-server) document states that "Remote server monitoring is not supported due to performance reasons." 

What alternative do you think I should implement to overwatch my remote server?

Thanks for all the work on this. Dokploy feels much intuitive and cleaner for a Frontend guy with minimal DevOps experience like me,

**Comment by rallisf1** [comment, marked as answer]
If you only need service and resource monitoring try [netdata](https://github.com/netdata/netdata)

If you need more customization and the ability to monitor other network devices try [zabbix](https://www.zabbix.com/)

**Reply by MuhammadM1998** [reply]
I thought that there's a way from within dokploy. Thanks I,ll propably use netdata
